ci/builder/Dockerfile

```
@@ -158,6 +159,10 @@ RUN pip3 install \
     -r /workdir/requirements-dev.txt \
     && rm /workdir/requirements*.txt
 
+RUN curl -fsSL https://github.com/deb-s3/deb-s3/releases/download/0.11.3/deb-s3-0.11.3.gem > deb-s3-0.11.3.gem \
+    && gem install ./deb-s3-0.11.3.gem
+
```

These two RUN commands should be merged, each separate one adds a small amount of time to docker image download/start time.

```suggestion
    -r /workdir/requirements-dev.txt \
    && rm /workdir/requirements*.txt \
    && curl -fsSL https://github.com/deb-s3/deb-s3/releases/download/0.11.3/deb-s3-0.11.3.gem > deb-s3-0.11.3.gem \
    && gem install ./deb-s3-0.11.3.gem

```